uebung3.py

```diff
@@ -0,0 +1,33 @@
+import math
+
+N = 3
+middle = (N*3)+2
+
+## Row 1
+print(" " * (middle-1), end="")
+print("y", end="")
+print(" " * (middle-1),end="\n\n")
+
+## Row 2
+print(" " * (middle-1), end="")
+print("^", end="")
+print(" " * (middle-1),end="\n")

```

Comment:
generally for things you do often like those three statements here you always wanna find a way to do it in one, e.g. by the use of this format string or by making it a function with a parameter